### DISCUSSION TITLE
how to set deafult as camera

### DISCUSSION THREAD

**Discussion by rddewan** [fluttercandies/flutter_wechat_assets_picker#362] [Q&A] [answered]
By default it's on Recent , is there a way to change it to Camera?
![Simulator Screen Shot - iPhone 11 Pro Max - 2022-09-14 at 14 00 00](https://user-images.githubusercontent.com/13164167/190084163-4435d5a0-55df-438e-a9b9-81f251b96c19.png)

**Comment by AlexV525** [comment, marked as answer]
Use `SortPathDelegate`.
